### PR TITLE
Streamline live view

### PR DESCRIPTION
**Pull request by NickM-27**

### CODE REVIEW

File: frigate/app.py [hunk 1]
```diff
@@ -647,6 +647,13 @@ def start(self) -> None:
 
         self.init_logger()
         logger.info(f"Starting Frigate ({VERSION})")
+
+        if not os.environ.get("I_PROMISE_I_WONT_MAKE_AN_ISSUE_ON_GITHUB"):
+            print(
+                "Frigate 0.14 UNSTABLE - not for public use at this time. Please use Frigate stable"
+            )
+            sys.exit(1)
+
```
Comment (skrashevich) [inline frigate/app.py:656]: 👍

**Comment by skrashevich** [comment]
PR #7071 is completely absorbed by this, ye?

**Comment by NickM-27** [comment]
> PR #7071 is completely absorbed by this, ye?

Yes. We are going to wait until things are more finalized before doing a clean sweep of issues and PRs

File: frigate/app.py (hunk 1 above)
Comment (xconverge) [reply frigate/app.py:656]: 😆 you guys just can't win

File: web/src/components/player/JSMpegPlayer.tsx [hunk 2]
```diff
@@ -34,6 +37,11 @@ export default function JSMpegPlayer({
   const scaledHeight = useMemo(() => {
     const scaledHeight = Math.floor(availableWidth / aspectRatio);
     const finalHeight = Math.min(scaledHeight, height);
+    console.log(`${containerWidth} / ${aspectRatio} -< ${containerHeight}`);
```
Comment (blakeblackshear) [inline web/src/components/player/JSMpegPlayer.tsx:40]:
```suggestion
```

Stray console log.

**Review by blakeblackshear — APPROVED** [review]

**Pull request merged**